File: app/src/main/java/me/rosuh/searchindouban/AboutPageDialog.java
package me.rosuh.searchindouban;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

import de.hdodenhof.circleimageview.CircleImageView;
/**
 * 这个类是用来显示 App 作者信息的；
 * 此类使用的是 Activity 继承 Dialog 的主题，来达到显示小窗口的目的
 * * @author rosuh 2018-4-24 10:40:23
 * @version 2.1
 * @since 1.8
 */

public class AboutPageDialog extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about_page_dialog);
        setTitle(R.string.about_page);
    }
}

File: app/src/main/java/me/rosuh/searchindouban/AboutPageDialog.kt
package me.rosuh.searchindouban

import android.os.Bundle
import android.support.v4.app.DialogFragment
import android.text.method.LinkMovementMethod
import android.view.Gravity
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView

/**
 * 这个类是用来显示 App 作者信息的；
 * * @author rosuh 2018-4-24
 */

class AboutPageDialog : DialogFragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.about_page_dialog, container, false)
        val tv = view.findViewById<TextView>(R.id.project_page_text_view)
        tv.movementMethod = LinkMovementMethod.getInstance()
        return view
    }

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)
        isCancelable = true
        val window = dialog.window
        window!!.setGravity(Gravity.CENTER)
        window.setWindowAnimations(R.style.animate_dialog)
//        window.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
        window.setLayout(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
    }
}
